### PR TITLE
PEP8 formatting

### PR DESCRIPTION
**Pull request by ghost**
import reordering and PEP-8 conventions, wrapped to 100 characters

### CODE REVIEW

**Comment by ghost** [comment]
changed wrapping to 79 characters, and reverted dictionary alignment

File: adcircpy/driver.py [hunk 1]
```diff
@@ -11,24 +11,24 @@
...
 
     def __init__(
-        self,
-        mesh: AdcircMesh,
-        start_date: datetime,
-        end_date: datetime,
-        spinup_time: timedelta = None,
-        netcdf: bool = True,
-        server_config: Union[int, SSHConfig, SlurmConfig] = None
+            self,
```
Comment (jreniel) [inline adcircpy/driver.py:25]: Why the extra indentation?

Comment (ghost) [reply adcircpy/driver.py:25]: from what I've seen in PEP8 [it wants extra indentation blocks in continuation and signatures](https://www.python.org/dev/peps/pep-0008/#indentation)

Comment (jreniel) [reply adcircpy/driver.py:25]: Ok, thanks for the clarification!

**Pull request merged**